hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/SstFilteringService.java

```diff
@@ -210,10 +217,6 @@ public BackgroundTaskResult call() throws Exception {
                 }
               }
             }
-            markSSTFilteredFlagForSnapshot(snapshotInfo.getVolumeName(),
-                snapshotInfo.getBucketName(), snapshotInfo.getName());
-            snapshotLimit--;
-            snapshotFilteredCount.getAndIncrement();
           } catch (RocksDBException | IOException e) {
             LOG.error("Exception encountered while filtering a snapshot", e);
```

Comment:
nit: should we log it at an info level, if SstFileringService fails because the snapshot is purged? Otherwise, it will be unnecessary noisy logs.

Reply:
You cannot be sure why the issue occured. RocksDbException can thrown for many reasons. One of them being the db being closed by another thread.